tests/OSPSuite.Presentation.Tests/Importer/Presenters/ImporterPresenterSpecs.cs

```diff
@@ -298,13 +298,21 @@ public void invokes_column_mapping_presenter()
 
    public class When_mapping_completed_with_loaded_data : concern_for_ImporterPresenter
    {
+      protected Cache<string, DataSheet> _sheets;
+
+      protected override void Context()
+      {
+         base.Context();
+         A.CallTo(() => _dataSource.ValidateDataSource(A<IReadOnlyList<ColumnInfo>>.Ignored, A<IDimensionFactory>.Ignored)).Returns(true);
+         _sheets = new Cache<string, DataSheet>();
+         _sheets.Add("sheet1", A.Fake<DataSheet>());
+      }
+
       protected override void Because()
       {
          base.Because();
```

Comment:
base.because should go. .I am going to merge this for now but please let's try to refactor the test so that the action being tested , is clear and does not require jumping around